### PR TITLE
[Ready] Icon clarity revision.

### PR DESCRIPTION
**Pull request by king5327**
Three purposes to this: have no identical icon forms (within reason, the head of a department can have an icon matching the icon of their "generic" subordinate rank), remove abstract lettering, and allow no non-bridge role to have a command blue icon. Icons changed:

Captain: Big letter C to Star (From TG)
HoP: Robotics figure to Big Head (From TG)
Internal Affairs: Letters IA to upper body and waist of humanoid figure. (Looks businessy)
Quartermaster: Cargotech in Blue to Small Head in Cargo Colour (From TG)
Paramedic: Similar to current SaR but without the red point (Becomes White Cross on Blue without corners)
Psychiatrist: Icon is now Question Mark (Dark on Light, as opposed to unknown's light on dark)
Visitor: Big letter V to Grey Box framed in White
Intern: Big letter I to Eye Form (Interns need to keep their eyes open)

Main expedition roles now have an internally consistent icon style: 
Explorer: Old icon (no idea what it was) to upward chevron.
Pathfinder: Old Explorer in Blue to upward chevron surmounting bar in Science Colour
Pilot: Vaguely plane-shaped symbol to downward chevron below bar.
Search and Rescue: Cross with red dot and missing corners to downward chevron.

### CODE REVIEW

**Comment by king5327** [comment]
For those of you about to complain about before/after images, head to the checks tab where IconDiffBot has them all shown.

**Comment by king5327** [comment]
How am I supposed to modularize a single DMI file change?

File: maps/tether/tether_defines.dm [hunk 1]
```diff
@@ -45,7 +45,7 @@
 
 	lobby_icon = 'icons/misc/title_vr.dmi'
 	lobby_screens = list("tether")
-	id_hud_icons = 'icons/mob/hud_jobs_vr.dmi'
```
Comment (deathride58) [inline maps/tether/tether_defines.dm]: comment isnt really necessary

Comment (king5327) [reply maps/tether/tether_defines.dm]: Is it wrong to tell the next guy that's going to change the code that that line is meaningless?

**Comment by king5327** [comment]
Updated to not break the offdutyco on test.

**Pull request merged**